### ISSUE TITLE
Script keeps running even after it's done

### ISSUE THREAD

**Issue by hillerstorm** [renovatebot/renovate#3644]
**What Renovate type are you using?**
Both CLI and latest Docker image

**Describe the bug**
Running either the docker image, the bot itself or renovate-config-validator does what it's supposed to but then stays alive forever due to an open timeout handle introduced in 16.13.0.

**Did you see anything helpful in debug logs?**
Used [mafintosh/why-is-node-running](https://github.com/mafintosh/why-is-node-running) to check which handles are open:
```
Johans-MBP-2:renovate hillerstorm$ why-is-node-running bin/config-validator.js ../Blk.Renovate/config.js 
probing program /Users/hillerstorm/src/renovate/bin/config-validator.js
kill -SIGUSR1 49414 for logging
Validating renovate.json
Validating config.js
OK
There are 2 handle(s) keeping the process running

# SIGNALWRAP
/Users/hillerstorm/.nvm/versions/node/v10.15.3/lib/node_modules/why-is-node-running/include.js:3 - process.on('SIGUSR1', function() { why() })

# Timeout
/Users/hillerstorm/src/renovate/lib/manager/travis/package.js:53 - setTimeout(generatePolicies, offsetmilliseconds);
```

**Additional context**
Running the latest code d665297, node version 10.15.3.
Reverting back to 37acd1a fixes the problem.

**Comment by rarkins** [comment]
Thanks for the bug report. I’ll fix this

**Comment by hillerstorm** [comment]
@rarkins `renovate-config-validator` is still broken even after `01b5ae2`, much less of a serious issue though :)

**Comment by ViceIce** [comment]
Seems the timeouts are not cleared from validator.

It's a regression from #3643, which should fixes #1269 .
